docs/docs/02-concepts/03-platform-providers.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Platform providers
+id: platform-providers
+description: Wing platform providers
+keywords: [Wing platform providers, provisioning engines, multi-cloud]
+---
+
+When working with the Wing programming language, an integral part of the compilation process is the use of platform providers. In essence, platform providers specify how and where your application is deployed. They determine both the cloud environment and the provisioning engine that the code will be deployed with.
+
+## Platforms
+
+You can view the list of available builtin platform providers with the `wing compile --help` command. Here is an example of the output:
```

Comment:
What's the difference between a platform and a platform provider?

This section is titled Platforms but it starts talking about platform providers so I was left a little confused

(also I'm wondering if the article title should be simplified to "Platforms")

Reply:
There is not a difference, one just sounds fancier 😅

No the real answer is I have been using them synonymously but there might be a distinction between a general platform which might just be a small platform that adds an aspect to a construct tree and a platform provider that must be the first platform that offers an App?

I dunno, need to shop around the verbiage. For now I think maybe just simplify the terms to just be "platform" I wrote most of this doc in a frenzied stupor 

@Chriscbr WDYT?

Reply:
I like the idea of using platform everywhere. We could also maybe say a "platform provider" is the implementing code for a platform in the Wing toolchain, if it seems useful to distinguishing between platform as a concept and platform as an artifact / shareable thing. But yeah even I'm not 100% sure 😅

Reply:
I actually think that a more accurate term is "platform provider" because this is basically just an adapter that connects wing apps to a platform.

But I think it's okay to omit the "provider" in most cases.

Basically I am saying the current terminology feels right.